CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ Unreleased
 -----------
 
 ### Improvements/Changes
-- Layout: Improve layout margins and typographical consistency across several content pages. (#5880, #5884, #5887, #5888)
+- Layout: Improve layout margins and typographical consistency across several content pages. (#5880, #5884, #5887, #5888, #5906, #5908)
 - Typography: Updated monospace font to Roboto Mono for consistency across login.gov sites. (#5891)
+- Multi-factor authentication: Add ability to opt numbers back in to receiving SMS that have replied "STOP" (#5894)
```

Comment:
quick note, this feature is disabled for now so we may want to slightly tweak this in the release notes

Reply:
Got it, thanks!